convert/convert.go

```diff
@@ -14,6 +14,13 @@ import (
 type Parameters struct {
 	Architectures []string `json:"architectures"`
 	VocabSize     uint32   `json:"vocab_size"`
+
+	LoraLayers     uint32 `json:"lora_layers"`
```

Comment:
This doesn't belong here. Instead probably create a separate AdapterParameters

Reply:
Yeah, I think that makes sense. I went back and forth on this one because llama.cpp dumps everything into the KVs for the adapters, including an entirely new copy of the tokens, but they don't need to be in there.